### DISCUSSION TITLE
Buy an offical app in the windows store or just a fake one?

### DISCUSSION THREAD

**Discussion by allenk** [sumatrapdfreader/sumatrapdf#3140] [General]
I bought a Sumatra PDF reader from windows store.
[Sumatra PDF Reading Editor](https://www.microsoft.com/store/productId/9NVZCVQ0PZL4)

But I realized that it might be a fake app :(
If it is a fake, maybe I should report it and prevent more people like me from being victimized!

**Comment by GitHubRulesOK** [comment]
https://www.sumatrapdfreader.org/download-free-pdf-viewer

**Reply by SadPencil** [reply]
> https://www.sumatrapdfreader.org/download-free-pdf-viewer

This reply does not answer the question at all.

Some softwares offcially offer a priced product at microsoft store to get some financial supports, e.g. WinSCP. 

It is important to figure out whether the one in microsft store is an official one. And the download page link does not answer to this.

**Comment by SadPencil** [comment]
https://www.microsoft.com/store/productId/9NVZCVQ0PZL4

This is the latest link that might be a fake one. The link you provided says 'Invalid product ID'.

**Reply by allenk** [reply]
> https://www.microsoft.com/store/productId/9NVZCVQ0PZL4
> 
> This is the latest link that might be a fake one. The link you provided says 'Invalid product ID'.

Yes, your link is correct.
